TODO.txt

```diff
@@ -1,66 +1,61 @@
 
-- attribute values are typed - check GG xml. check ncdf api.
-- sql support in attributes
+Prototype demonstrate end-to-end filter to subset netcdf generation, with streaming on prod data.
 
-- interface and single method to communicate with server. 
-- cql filter expression parsing
-- wkt
+-----
+TODO
+- attribute values are typed - check GG xml. check ncdf api.
+- sql lookup for global attributes
 - global attributes and sql lookup.
-
 - need float and string literal types.
 
 -----
+TODO - OLD
 
-Important - We don't need to get rid of the example parsers. just ensure it's puggable 
-
-----
-prototype demonstrate end-to-end filter to subset netcdf generation, with streaming 
-on prod data.
-
-----
-PRIORITY
-- move connection pool in the unit tests.
+- postgres - parametize query parameters, rather than use text, injection attack
```

Comment:
Fair enough, but what you note here is exactly what a library such as JOOQ takes care of (and it's already been written and tested - the main advantage).